crates/client/rpc/src/versions/v0_8_0/methods/ws/lib.rs

```diff
@@ -0,0 +1,398 @@
+use std::borrow::Cow;
+
+use crate::{
+    errors::{StarknetWsApiError, WsResult},
+    versions::v0_8_0::StarknetWsRpcApiV0_8_0Server,
+};
+
+use super::BLOCK_PAST_LIMIT;
+
+#[jsonrpsee::core::async_trait]
+impl StarknetWsRpcApiV0_8_0Server for crate::Starknet {
+    async fn subscribe_new_heads(
```

Comment:
This is already handled [here](https://github.com/madara-alliance/madara/blob/0b0cf309e31b7cde6f533b9d5c693bd25f02e106/crates/client/rpc/src/errors.rs#L237) but I think I'll more this code to the init of `InternalError` instead of the `message` function.